### PR TITLE
Remove and Add Executors for Will Transfers

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
 - bcgov/entity#16290

*Description of changes:*
- Add ability to remove and add Executors
- Add unit tests

Screenshot 2023-05-26 at 4 06 31 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1358-gx0pulnl.web.app

File: ppr-ui/src/composables/mhrInformation/useTransferOwners.ts [hunk 1]
```diff
@@ -376,7 +376,14 @@ export const useTransferOwners = (enableAllActions: boolean = false) => {
     // check if supportingDocument is either Death Certificate or Grant of Probate
     hasValidSupportDocs: (owner: MhrRegistrationHomeOwnerIF): boolean => {
       if (owner.action === ActionTypes.ADDED) return true
+
+      // if deleted Exec, Admin or Trustee, then docs are not required and are valid
+      if (owner.action === ActionTypes.REMOVED &&
+        [HomeOwnerPartyTypes.EXECUTOR, HomeOwnerPartyTypes.ADMINISTRATOR, HomeOwnerPartyTypes.TRUSTEE]
```
Comment (orelbn) [inline ppr-ui/src/composables/mhrInformation/useTransferOwners.ts:382]: We should consider adding `isPartyTypeEAT` to useHomeOwners as its used a few times throughout the code

**Review by orelbn — APPROVED** [review]
Nice job! tests are looking good.

**Review by cameron-eyds — APPROVED** [review]

**Pull request merged**